packages/openapi-parser/src/utils/upgrade-from-three-to-three-one.ts

```diff
@@ -121,16 +121,21 @@ const applyChangesToDocument = (schema: UnknownObject, path: string[]) => {
     return {}
   }
 
+  // 6. Handle older formats
+  const { format: _, ...rest } = schema
```

Comment:
Is this actually valid though? Are old formats able to be used 1:1? If we do proceed we should probably put a specific JSON schema version locally or at the top of the document.